temba_client/v2/__init__.py

```diff
@@ -131,19 +131,22 @@ def get_classifiers(self, uuid=None):
         params = self._build_params(uuid=uuid)
         return self._get_query("classifiers", params, Classifier)
 
-    def get_contacts(self, uuid=None, urn=None, group=None, deleted=None, before=None, after=None):
+    def get_contacts(self, uuid=None, urn=None, group=None, deleted=None, reverse=None, before=None, after=None):
```

Comment:
I would add it last.. just in case someone is passing all arguments positionally and also it feels like it belongs last

Reply:
Ok